echopype/utils/common.py

```diff
@@ -34,3 +39,50 @@ def swap_dims_channel_frequency(ds: xr.Dataset) -> xr.Dataset:
             "Duplicated transducer nominal frequencies exist in the file. "
             "Operation is not valid."
         )
+
+
+def add_location(ds: xr.Dataset, echodata: EchoData = None, nmea_sentence: Optional[str] = None):
```

Comment:
Should this be in the `preprocess` module instead? After all it enhances the spatial coherence of the data? 😄 

https://github.com/OSOceanAcoustics/echopype/blob/12693a72badcf54a18408e694b72e7b0e6d48023/echopype/preprocess/api.py#L2

Reply:
Oh that is very true! I agree with this and will move it.

Reply:
I guess I had forgotten that I've written those sentences...